github-rate-limits.py

```diff
@@ -1,14 +1,53 @@
 #!/usr/bin/env python
 from __future__ import print_function
-from github import Github
-from os.path import expanduser
+import json, os
 from datetime import datetime
+from hashlib import sha1
+from os.path import expanduser
 from socket import setdefaulttimeout
+from github import Github
+from es_utils import send_payload
 
 setdefaulttimeout(120)
 
 if __name__ == "__main__":
     gh = Github(login_or_token=open(expanduser("~/.github-token")).read().strip())
-    print("API Rate Limit")
-    print("Limit, Remaining: ", gh.rate_limiting)
-    print("Reset time (GMT): ", datetime.fromtimestamp(gh.rate_limiting_resettime))
+    print("Checking GitHub API Rate Limit")
+    remaining, limit = gh.rate_limiting
+    print("Remaining calls: ", remaining)
+    print("Max calls: ", limit)
+    reset_time = datetime.fromtimestamp(gh.rate_limiting_resettime)
+    print("Reset time (GMT): ", reset_time)
+
+    JENKINS_PREFIX = "jenkins"
+    try:
+        JENKINS_PREFIX = os.environ["JENKINS_URL"].strip("/").split("/")[-1]
+    except:
+        JENKINS_PREFIX = "jenkins"
+
```

Comment:
remove `JENKINS_PREFIX` , it is not needed for monitoring the gh api rate

Reply:
I do not think `JENKINS_PREFIX` has any value for monitoring the api rate. Instead you should use github user name. So that if in future we have to monitor api rate for another user then we can make use of that.

Reply:
It makes sense, thanks!